crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/dto/AssessmentRequestDTO.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.laa.crime.contribution.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class AssessmentRequestDTO {
+
+    private String iojResult;
```

Comment:
Is there a reason why we are not modelling these as enums? It looks as though each field has a fixed set of possible values it can take, this is effectively the definition of an enum.
This would also have the added benefit of Spring validating these when the request hits the endpoint.

Reply:
Elsewhere application we used same approach and I think is not required in getAssessmentResult